### DISCUSSION TITLE
Your plugin with HOOBS

### DISCUSSION THREAD

**Discussion by sstracy** [d0n4v4nb3ck3r/homebridge-honeywell-home#703] [Ideas]
I know you have said to others that you are not developing for HOOBS but it seems like we are just missing a small setup step while trying to setup your great plugin with HOOBS but then after that, it would work fine? I'm wondering how willing you'd be into looking into this? If you wanted to be compensated some, would that help? Fact is, there are lots of people using Homebridge with HOOBS these days and it's probably going to keep getting more popular.

Please let me know your thoughts and thanks!

**Comment by donavanbecker** [comment]
I haven't really seen anything new coming out of HOOBs these days, so is it even worth it. Plus Homebridge UI has the power of the custom UI that allows the pairing of this plugin with Honeywell really easy. HOOBs doesn't have this.

I think more people are using Homebridge and switching back to regular homebridge then using HOOBs.

**Reply by sstracy** [reply]
Okay, thanks for the quick reply! I'll keep mulling over this one.

**Reply by donavanbecker** [reply]
> Okay, thanks for the quick reply! I'll keep mulling over this one.

I wouldn't even know where to start to support their UI.
